### ISSUE TITLE
Revisit whether we can purge the 'Read more' behaviour on Posts

### ISSUE THREAD

**Issue by stevejalim** [mdn/developer-portal#1502]
The pre-redesign code (HTML, CSS, Python) remains, but it's unclear whether we need it.

AC 
- [ ] Rationale confirmed for the existence (or not) of 'Read more' on posts published within the portal
- [ ] Code removed or commented to clarify, as appropriate

**Comment by valgrimm** [comment]
I'd say keep "Read More"

**Comment by stevejalim** [comment]
Ah, I've just noticed that if related links are set on a page (the "Read more") section, nothing gets rendered.

I think this is because the redesign either did not show Related Links, or I simply missed it PLUS the fact that there is a section of template that renders 'Related Posts' which is automatically populated from other things, which we don't want to happen.

Given there are no CMS-hosted Posts / Articles in production at the mo, this is not a criticial issue, but I think I should fix it up soon.

To that end: @valgrimm  am I right in assuming that you (like me) thought that a 'Read more' section would show a line of cards, based on the links specified in "Related Links", ie:
![Screenshot 2020-06-18 at 12.10.18.png](https://images.zenhubusercontent.com/5d7b94dd167772000187fd33/e482357e-75df-4551-8567-7ed85ad6d064)

If so, we'll also need to add support for selecting an image to use with that link (which is fine to do)

Let me know and I'll spin out a new card for the update work and close this one

**Comment by valgrimm** [comment]
> To that end: @valgrimm am I right in assuming that you (like me) thought that a 'Read more' section would show a line of cards, based on the links specified in "Related Links
Yes

> If so, we'll also need to add support for selecting an image to use with that link (which is fine to do)
> Let me know and I'll spin out a new card for the update work and close this one
The image to use with each link should be the card image. Or did I understand that correctly?

**Comment by stevejalim** [comment]
> The image to use with each link should be the card image. Or did I understand that correctly?

At the moment, the links are more like the free-form "link to anything" ones we have in streamfields (indeed, I've just noticed, "related links" is a just a streamfield that only allows simple links, because that's all the old design needed). 

I can refactor it to only use links to internal or external pages, so then we can use the card image associated with those pages, yeah.

**Comment by valgrimm** [comment]
> I can refactor it to only use links to internal or external pages, so then we can use the card image associated with those pages, yeah.

Yeah, that would be more consistent as the Read More would be inherently related to internal or external posts or videos page.

**Comment by stevejalim** [comment]
Thanks - sounds good. 

Queueing this up behind/ for after Search, as we're currently not using the Read More feature, unless you need it ASAP.
